assets/js/spoiler.js

```diff
@@ -9,28 +9,29 @@
 
     function buildSpoilers($elem) {
         $elem.each(function() {
-            $(this).addClass("spoiler-build");
-            $(this).before($("<a/>", {
-                text: "Afficher/Masquer le contenu masqué",
-                class: "spoiler-title ico-after view",
-                href: "#",
-                click: function(e) {
-                    $(this).next(".spoiler").toggle();
-                    e.preventDefault();
-                }
-            }));
+            var $this = $(this);
+
+            if(!$this.hasClass("spoiler-build")) {
+                $this.before($("<a/>", {
+                    text: "Afficher/Masquer le contenu masqué",
+                    class: "spoiler-title ico-after view",
+                    href: "#",
+                    click: function(e) {
+                        $(this).next(".spoiler").toggle();
```

Comment:
Afin d'être cohérent, il ne faudrait pas écrire $this.next au lieu de $(this).next ?